source/geometry_model/chunk.cc

```diff
@@ -167,6 +167,14 @@ namespace aspect
       point1_lon = p1_lon;
     }
 
+    template <int dim>
+    std::unique_ptr<Manifold<dim,dim>>
+    Chunk<dim>::ChunkGeometry::
+    clone() const
+    {
+        return std_cxx14::make_unique<ChunkGeometry>();
```

Comment:
That satisfies the compiler, but it's not a copy of the current object. You'll have to somehow set the contents of the chunk geometry object to be a copy of the current object.